tests/cases/standalone/common/types/decimal/decimal_aggregates.result

```diff
@@ -3,9 +3,9 @@
 SELECT arrow_typeof(FIRST_VALUE('0.1'::DECIMAL(4,1)));
 
 +----------------------------------------+
-| arrow_typeof(FIRST_VALUE(Utf8("0.1"))) |
+| arrow_typeof(first_value(Utf8("0.1"))) |
 +----------------------------------------+
-| Decimal128(4, 1)                       |
+| Float64                                |
```

Comment:
The type of a decimal is float64 😅

Reply:
decimal is not one of the primitive `NUMERICS`. It got cast to the nearest float64